specs/disciplina/educator/api/educator.yaml

```diff
@@ -657,6 +659,112 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrResponse'
+
+  /certificates:
+    get:
+      tags:
+        - Certificates
+      summary: Get the list of certificates created by Educator
+      description: >-
+        Gets all the certificates created by Educator. Each entry contains
+        certificate metadata and certificate ID.
+      operationId: getCertificates
+      parameters:
+        - $ref: '#/components/parameters/Offset'
+        - $ref: '#/components/parameters/Limit'
+        - $ref: '#/components/parameters/Order'
+        - $ref: '#/components/parameters/OnlyCount'
+      responses:
+        200:
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                type: object
+                required:
+                  - count
+                properties:
+                  count:
+                    $ref: '#/components/schemas/Count'
+                  certificates:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/Certificate'
```

Comment:
Meh, so eventually we store all those fields in certificates in separate columns, not like compiled PDF. Sad (but fair I guess)